### PR TITLE
Homework schedule

### PR DESCRIPTION
**Pull request by lukasz-kukulka**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: schedule
🏅 Granted 12 XP for @lukasz-kukulka

**Comment by ziobron** [comment]
⏱ 3 XP extra for @lukasz-kukulka for on-time delivery

**Comment by ziobron** [comment]
✅ CI Passed - Task: schedule-bonus
🏅 Granted 10 XP for @lukasz-kukulka

**Comment by ziobron** [comment]
⏱ 3 XP extra for @lukasz-kukulka for on-time delivery

File: homework/schedule/schedule.hpp [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+void schedule(std::function<void (std::string, double)> func, std::chrono::seconds duration, std::string text, double value) {
+    std::this_thread::sleep_for(duration);
+    func(text, value);
+}
+
+template <typename T, typename... Args>
+void schedule(T func, std::chrono::seconds duration, Args... args) {
+    std::this_thread::sleep_for(std::chrono::seconds(duration));
+    func (args...);
```
Comment (nauka-programowania-MA) [inline homework/schedule/schedule.hpp:26]: use perfect forwarding here. std::forward<Args>(args)...

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**